libloki/api.js

```diff
@@ -115,16 +115,13 @@
     // Serialise array of byteBuffers into 1 byteBuffer
     const byteBuffer = serialiseByteBuffers(contactDetails);
     const data = new Uint8Array(byteBuffer.toArrayBuffer());
-    const contacts = new textsecure.protobuf.SyncMessage.Contacts({
+    return new libsession.Messages.Outgoing.ContactSyncMessage({
```

Comment:
should this message take in `rawContacts` instead and we can build the protobuf inside it?

Reply:
best would be for it to take the `sessionContacts` directly, I will check what I can do

Reply:
Well not necessarily, a `RawContact` can just be the information needed for contact sync, the message shouldn't need to worry about how to extract that information from `sessionContacts`